apps/web/src/app/(profile)/[username]/page.tsx

```diff
@@ -22,7 +22,7 @@ import { getProgressData, getUserActivity } from './user-info';
 import { auth } from '~/server/auth';
 import { MagicIcon } from '@repo/ui/components/magic-icon';
 import { getBadges } from '~/app/actions/badges/_actions';
-import { aotBadgeKeys } from '~/app/actions/badges/badge_types/advent_badges';
+import { aotBadgeKeys } from '~/app/actions/badges/badge_types/advent-badges';
```

Comment:
kabab the `badge_types` folder as well 😏

Reply:
its now kebabed, good catch.